crates/generate-json-rpc-spec/Cargo.toml

```diff
@@ -25,4 +25,7 @@ sui-types = { path = "../sui-types" }
 sui-config = { path = "../sui-config" }
 test-utils = { path = "../test-utils" }
 hyper = { version = "0.14.18", features = ["full"] }
+
+
+[target.'cfg(not(windows))'.dependencies]
```

Comment:
All of these should not be needed if the deps in workspace hack are only for not(windows). If possible let's drop these changes in all of our crates tomls.

Reply:
Cool I will do this and re-test (most likely tomorrow)